### DISCUSSION TITLE
how to setup a chartflow to "initialize, setup" a pinecone or vector db independent to a chain/agent

### DISCUSSION THREAD

**Discussion by Kvkthecreator** [FlowiseAI/Flowise#428] [Q&A] [answered]
(potential silly question), but i want to initialize my database (ie. pinecone) of namespaces, metadata separately from a chain/agent chartflow.
reason is to separate the namespace/metadata creation in pinecone independent to chatflow (my logic is, so a user can "choose a agent or chartflow") and then separately start a conversation.

**Comment by HenryHengZJ** [comment, marked as answer]
this is currently not possible as every flow ends with a chain/agent which you can interact with.
having said that, we have plan to introduce **Workflow** which allow you to just construct the flow without needing to have chain/agent as the last node

**Reply by Kvkthecreator** [reply]
thanks for reply! regardless its informative to know what is/isn't possible. look forward to new workflows when they'll be available :) thank you
